asn.c

```diff
@@ -17,8 +18,20 @@
    along with this program.  If not, see <http://www.gnu.org/licenses/>.
 */
 
+#include <gnutls/gnutls.h>
+#include <libtasn1.h>
+#include <stdlib.h>
+
+#include "asn.h"
+
 #include "rdesktop.h"
+// Generated by asn1Parser
+#include "pkix_asn1_tab.c"
```

Comment:
This file isn't included in this commit, so it will create a broken commit. Please merge this commit and the next one.

Also, we should probably have a configure test for libtasn1?